socorro/unittest/signature/test_rules.py

```diff
@@ -221,6 +221,22 @@ def test_generate_4(self):
         sig, notes = s.generate(a)
         assert sig == 'f | e | d | i'
 
+    def test_specifiers_and_return_types(self):
+        """prefix and irrelevant should match ignoring specifiers and return types"""
+        generator = self.setup_config_c_sig_tool(
+            # irrelevant
+            ig=['ignored'],
+            # prefix
+            pr=['pre1', 'pre2'],
+        )
+        source_list = (
+            'static void ignored',
+            'pre1',
+            'static bool pre2',
```

Comment:
Should we add another source in the list after `pre2`? That'd help check that `pre2` is actually being treated as a prefix despite having the `static bool` in front. Right now it's not clear from the assertion that `pre2` matched the prefix list vs just being the first unmatched source.

Reply:
This is why I :heart: your reviews.